EssentialFeed/EssentialFeedTests/Image Comment/Presentation/ImageCommentsPresenterTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import XCTest
+import EssentialFeed
+
+class ImageCommentsPresenterTests: XCTestCase {
+	func test_title_isLocalized() {
+		XCTAssertEqual(ImageCommentsPresenter.title, localized("IMAGE_COMMENTS_TITLE"))
+	}
+
+	func test_map_createsViewModel() {
+		let now = Date()
+		let calendar = Calendar.current
+		let locale = Locale.current
```

Comment:
You could inject an English locale (e.g., `en_US_POSIX`), so the test will not rely on global environment details that might change. Otherwise, the test may fail if running in a different locale because it's expecting the English relative date strings, e.g., `1 minute ago`. 

```suggestion
		let locale = Locale(identifier: "en_US_POSIX")
```